README.md

```diff
@@ -2,388 +2,24 @@
-
-Please note that only Docker is officially supported as a backend by the Dev Container extension. <sup>[1](https://code.visualstudio.com/remote/advancedcontainers/docker-options)</sup>
-
-## Telemetry
-
-On its own, the Ruby LSP does not collect any telemetry by default, but it does support hooking up to a private metrics
-service if desired.
-
-In order to receive metrics requests, a private plugin must export the `ruby-lsp.getPrivateTelemetryApi` command, which
-should return an object that implements the `TelemetryApi` interface defined
-[here](https://github.com/Shopify/vscode-ruby-lsp/blob/main/src/telemetry.ts).
-
-Fields included by default are defined in `TelemetryEvent`
-[here](https://github.com/Shopify/vscode-ruby-lsp/blob/main/src/telemetry.ts). The exported API object can add any
-other data of interest and publish it to a private service.
-
-For example,
-
-```typescript
-// Create the API class in a private plugin
-class MyApi implements TelemetryApi {
-  sendEvent(event: TelemetryEvent): Promise<void> {
-    // Add timestamp to collected metrics
-    const payload = {
-      timestamp: Date.now(),
-      ...event,
-    };
-
-    // Send metrics to a private service
-    myFavouriteHttpClient.post("private-metrics-url", payload);
-  }
-}
-
-// Register the command to return an object of the API
-vscode.commands.registerCommand(
-  "ruby-lsp.getPrivateTelemetryApi",
-  () => new MyApi(),
-);
-```
-
-## Formatting
-
-When `rubyLsp.formatter` is set to `auto`, Ruby LSP tries to determine which formatter to use.
-
-If the bundle has a **direct** dependency on a supported formatter, such as `rubocop` or `syntax_tree`, that will be used.
-Otherwise, formatting will be disabled and you will need add one to the bundle.
-
-## Contributing
-
-Bug reports and pull requests are welcome on GitHub at https://github.com/Shopify/vscode-ruby-lsp.
-This project is intended to be a safe, welcoming space for collaboration, and contributors
-are expected to adhere to the
-[Contributor Covenant](https://github.com/Shopify/vscode-ruby-lsp/blob/main/CODE_OF_CONDUCT.md)
-code of conduct.
-
-Before contributing, please make sure to [sign the Contributor License Agreement](https://cla.shopify.com).
-
-### Debugging
-
-Interactive debugging works for both running the extension or tests. In the debug panel, select whether to run the
-extension in development mode or run tests, set up some breakpoints and start with F5.
-
-### Tracing LSP requests and responses
-
-LSP server tracing (logging) can be controlled through the `ruby lsp.trace.server` config key in the `.vscode/settings.json`
-config file.
-
-Possible values are:
-
-- `off`: no tracing
-- `messages`: display requests and responses notifications
-- `verbose`: display each request and response as JSON
+This repository has moved! Find the Ruby LSP VS Code extension in our
+[monorepo](https://github.com/Shopify/ruby-lsp/releases/tag/vscode-ruby-lsp-v0.5.12).
```

Comment:
Let's make this more visible:

```suggestion
> [!IMPORTANT]  
> This repository has moved! Find the Ruby LSP VS Code extension in our
> [monorepo](https://github.com/Shopify/ruby-lsp/releases/tag/vscode-ruby-lsp-v0.5.12).
```

Reply:
Yeah, indeed. Honestly, I thought I had linked to the `vscode` directory, but I must have pasted the wrong link.